omniscidb/QueryEngine/Optimization/AnnotateInternalFunctionsPass.h

```diff
@@ -64,11 +76,9 @@ class AnnotateInternalFunctionsPass : public llvm::CallGraphSCCPass {
       }
     }
 
-    return updated_function_defs;
+    return llvm::PreservedAnalyses::all();
```

Comment:
The change probably breaks some analyses, we should double-check it.

Reply:
With the new PM, I need to return PA or PreservedAnalyses::none(); or llvm::PreservedAnalyses::all(); for example

Reply:
`llvm::PreservedAnalyses` represents a set of analyses that are preserved by a transformation pass. You can create such an object and specify which analyses are preserved/abandoned. The `none()` and `all()` are just convenience methods.

Reply:
I see. We would need to check how to test if the analyses are broken or not.